x-pack/plugins/security_solution/public/explore/index.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Storage } from '@kbn/kibana-utils-plugin/public';
+import type { AnyAction, Reducer } from 'redux';
+import type { HostsState } from './hosts/store';
+import type { UsersState } from './users/store';
+import { TableId } from '../../common/types';
+import type { SecuritySubPluginWithStore } from '../app/types';
+import { routes } from './routes';
+import type { NetworkState } from './network/store';
+import { initialNetworkState, networkReducer } from './network/store';
+import { getDataTablesInStorageByIds } from '../timelines/containers/local_storage';
+import { initialUsersState, usersReducer } from './users/store';
+import { hostsReducer, initialHostsState } from './hosts/store';
+
+export interface ExploreState {
+  network: NetworkState;
+  hosts: HostsState;
+  users: UsersState;
+}
+
+export interface ExploreReducer {
+  network: Reducer<NetworkState, AnyAction>;
+  hosts: Reducer<HostsState, AnyAction>;
+  users: Reducer<UsersState, AnyAction>;
+}
+
+export class Explore {
```

Comment:
@YulNaumenko @semd this is the part that needs some opinion. I added `exploreDataTables` to make typing this easier. Note that users never utilized the local storage table feature, should it be there? Not sure I totally understand/remember this feature

Reply:
I don't think we have the local storage cleanup or utilize functionality, but we have a migration logic:
https://github.com/elastic/kibana/blob/5a9c04b24fea2dde5858d8609d53d0baf1e82753/x-pack/plugins/security_solution/public/timelines/containers/local_storage/index.tsx#L60-L61
Do you think this is something you can use for your case?